phpunit.xml.dist

```diff
@@ -5,6 +5,7 @@
 	convertErrorsToExceptions="true"
 	convertNoticesToExceptions="true"
 	convertWarningsToExceptions="true"
+	beStrictAboutTestsThatDoNotTestAnything="false"
```

Comment:
Is there any way to be less heavy-handed? I think this rule could still be useful in general, but I agree we shouldn't warn about the abstract methods.